frontend/.gitignore

```
@@ -32,3 +32,7 @@ pnpm-lock.yaml
 
 server.crt
 server.key
+
+test-results/
+
+results/
```

is this what you are going for?
it might not be a good idea to ignore `directories` on a `global level` (in frontend folder)

```suggestion
/test-results
/tests/results
```